src/main/java/com/checkout/payments/apm/OxxoSource.java

```diff
@@ -0,0 +1,30 @@
+package com.checkout.payments.apm;
+
+import com.checkout.common.CountryCode;
+import com.checkout.common.PaymentSource;
+import com.checkout.payments.RequestSource;
+import com.google.gson.annotations.SerializedName;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public final class OxxoSource implements RequestSource {
+
+    private final PaymentSource type = PaymentSource.OXXO;
+
+    @SerializedName("integration_type")
+    private final IntegrationType integrationType = IntegrationType.REDIRECT;
+
+    private final CountryCode country;
```

Comment:
In general please make request classes attributes non-final. The `type` can be final, of course.